ui/admin/app/components/auth-and-model-providers/constants.ts

```diff
@@ -26,60 +27,73 @@ export const ModelProviderLinks = {
-		"Google Credentials JSON":
-			"Google Cloud Account Credentials - JSON File Contents: https://cloud.google.com/iam/docs/keys-create-delete#creating",
-		"Google Cloud Project": "Google Cloud Project ID",
-	},
+	// OpenAI
+	OBOT_OPENAI_MODEL_PROVIDER_API_KEY:
+		"OpenAI API Key. Can be created and fetched from https://platform.openai.com/settings/organization/api-keys or https://platform.openai.com/api-keys",
+
+	// Azure OpenAI
+	OBOT_AZURE_OPENAI_MODEL_PROVIDER_ENDPOINT:
+		"Endpoint for the Azure OpenAI service (e.g. https://<resource-name>.<region>.api.cognitive.microsoft.com/)",
+	OBOT_AZURE_OPENAI_MODEL_PROVIDER_CLIENT_ID:
+		"Unique identifier for the application when using Azure Active Directory. Can typically be found in App Registrations > [application].",
+	OBOT_AZURE_OPENAI_MODEL_PROVIDER_CLIENT_SECRET:
+		"Password or key that app uses to authenticate with Azure Active Directory. Can typically be found in App Registrations > [application] > Certificates & Secrets",
+	OBOT_AZURE_OPENAI_MODEL_PROVIDER_TENANT_ID:
+		"Identifier of instance where the app and resources reside. Can typically be found in Azure Active Directory > Overview > Directory ID",
+	OBOT_AZURE_OPENAI_MODEL_PROVIDER_SUBSCRIPTION_ID:
+		"Identifier of user's Azure subscription. Can typically be found in Azure Portal > Subscriptions > Overview.",
+	OBOT_AZURE_OPENAI_MODEL_PROVIDER_RESOURCE_GROUP:
+		"Container that holds related Azure resources. Can typically be found in Azure Portal > Resource Groups > [OpenAI Resource Group] > Overview",
+
+	// Ollama
+	OBOT_OLLAMA_MODEL_PROVIDER_HOST:
+		"IP Address for the ollama server (eg. 127.0.0.1:1234)",
+
+	// Groq
+	OBOT_GROQ_MODEL_PROVIDER_API_KEY:
+		"Groq API Key. Can be created and fetched from https://console.groq.com/keys",
+
+	// vLLM
+	OBOT_VLLM_MODEL_PROVIDER_ENDPOINT:
+		"Endpoint for the vLLM OpenAI service (eg. http://localhost:8000)",
+	OBOT_VLLM_MODEL_PROVIDER_API_KEY:
+		"VLLM API Key set when starting the vLLM server",
+
+	// DeepSeek
+	OBOT_DEEPSEEK_MODEL_PROVIDER_API_KEY:
+		"DeepSeek API Key. Can be created and fetched from https://platform.deepseek.com/api_keys",
+
+	// Anthropic Bedrock
+	OBOT_ANTHROPIC_BEDROCK_MODEL_PROVIDER_ACCESS_KEY_ID: "AWS Access Key ID",
+	OBOT_ANTHROPIC_BEDROCK_MODEL_PROVIDER_SECRET_ACCESS_KEY:
+		"AWS Secret Access Key",
+	OBOT_ANTHROPIC_BEDROCK_MODEL_PROVIDER_SESSION_TOKEN: "AWS Session Token",
+	OBOT_ANTHROPIC_BEDROCK_MODEL_PROVIDER_REGION:
+		"AWS Region - make sure that the models you want to use are available in this region: https://docs.aws.amazon.com/bedrock/latest/userguide/models-regions.html",
+
+	// Gemini Vertex
+	OBOT_GEMINI_VERTEX_MODEL_PROVIDER_GOOGLE_CREDENTIALS_JSON:
+		"Google Cloud Account Credentials - JSON File Contents: https://cloud.google.com/iam/docs/keys-create-delete#creating",
+	OBOT_GEMINI_VERTEX_MODEL_PROVIDER_GOOGLE_CLOUD_PROJECT:
+		"Google Cloud Project ID",
+
+	// Generic OpenAI
+	OBOT_GENERIC_OPENAI_MODEL_PROVIDER_BASE_URL:
```

Comment:
Nit: probably better to have some generic placeholder for api url like `https://{your_openai_server_hostname}/v1` than localhost?

Reply:
This is actually the URL for LM-Studio 😁
I will adjust this while doing the metadata change